leetcode/src/190.c

```diff
@@ -0,0 +1,10 @@
+uint32_t reverseBits(uint32_t n) {
+    uint TotalBits = 32;
+    uint32_t reverse_int = 0;
+    uint i;
+    for(i = 0; i < TotalBits; i++) {
+        if((n & (UINT32_C(1) << i)))
```

Comment:
Could you put some comment for explanation? Ex: `check bit is 1 or 0`

Reply:
Done. Please take a look